service/hub/server/config.go

```diff
@@ -82,6 +102,8 @@ func LoadConfig(fileName string) (*Config, error) {
 	if err = validate(c); err != nil {
 		return nil, fmt.Errorf("invalid config: %s", err)
 	}
+	// store client globally
+	k8sClient, _ = kubernetes.NewKubernetesClient(c.Server.Kubeconfig, "")
```

Comment:
split cheaker and var initer.

Reply:
Why reassign global k8sClient to new user spec kubeconfig?

Reply:
deleted.